src/Reporting/src/App.Metrics.Formatting.StatsD/MetricsStatsDStringOutputFormatter.cs

```diff
@@ -63,5 +65,18 @@ public MetricsStatsDStringOutputFormatter(MetricsStatsDOptions options)
             await using var writer = new MetricSnapshotStatsDStringWriter(output, _samplers, _options);
             serializer.Serialize(writer, metricsData, MetricFields);
         }
+
+        public Task<List<string>> WriteAsync(MetricsDataValueSource metricsData, CancellationToken cancellationToken = default)
+        {
+            var writer = new MetricSnapshotStatsDStringWriter(null, _samplers, _options);
+            var serializer = new MetricSnapshotSerializer();
+            serializer.Serialize(writer, metricsData, MetricFields);
+
+            var result = writer.Sampler.Points
```

Comment:
Need to add some type of chunking here, so we don't go overboard and send every metric in its own datagram.